ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -516,6 +516,35 @@ final class Metrics(val registry: MetricRegistry) {
       val stateUpdateProcessing: Timer = registry.timer(Prefix :+ "processed_state_updates")
     }
 
+    // TODO append-only: streamline metrics upon cleanup
+    object parallelIndexer {
+      private val Prefix: MetricName = daml.Prefix :+ "parallel_indexer"
+
+      // bundle of metrics coming from instrumentation of the underlying thread-pool
+      val ingestionExecutor: MetricName = Prefix :+ "ingestion_executor"
+
+      // bundle of metrics coming from instrumentation of the underlying thread-pool
+      val inputMappingExecutor: MetricName = Prefix :+ "input_mapping_executor"
+
+      // the latency, which during an update element is residing in the mapping-stage (since batches are involved, this duration is divided by the batch size)
+      val inputMappingStageDuration: Timer =
+        registry.timer(Prefix :+ "input_mapping_stage_duration")
```

Comment:
Separate metrics segments translate on presentation platforms to very deep trees. I would suggest removing all the words that are technology specific and don't add any semantic context.
In this case i would just say "input_mapping"

Reply:
I am not sure what you mean...AFAIK in grafana it is not causing any issues, at least there is no deep trees formed from underscore separated strings. Are you referring to VMBC related metrics visualisation?
Just for reference: looking around in Metrics this is not really seem to be a rule... is this a new requirement?

Reply:
Yes it is mostly about wavefront. Every time there is an underscore, it creates a branch in the metrics tree name. You quickly learn to be frugal when you interact with a system like that. We did have a go once trimming long metric names. It got a bit worse since then.

Reply:
In wavefront this metrics will be represented as:
com.daml.parallel.indexer.input.mapping.count
com.daml.parallel.indexer.input.mapping.min
com.daml.parallel.indexer.input.mapping.mean
com.daml.parallel.indexer.input.mapping.max
com.daml.parallel.indexer.input.mapping.0.75
etc